src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -1200,6 +1204,37 @@ public void action() {
         }
     }
 
+    private class OpenShortScienceAction implements BaseAction {
+        private static final String BASIC_SEARCH_URL = "https://www.shortscience.org/internalsearch";
+
+        @Override
+        public void action() {
+            final List<BibEntry> bes = mainTable.getSelectedEntries();
```

Comment:
Please implement a Fetcher (as the DOIFetcher or other fetchers do). --> Then, you can easily write test cases for the different shortscience urls etc.

Reply:
But this is not a fetcher, or I'm mistaken?

Reply:
This isn't really a fetcher as I've understood things, as it is essentially the same as a OpenURLAction with the exception that the link has to be generated. It doesn't fetch any information from the web, but instead opens the user's web browser.